### PR TITLE
Remove cross app imports pt1

### PR DESCRIPTION
**Pull request by adamwhitlock1**
## Summary

- The first step in isolating the personalization apps is to remove the cross app imports in a few places
- For the view-dependents application, there were some imports from the static-pages that were previously behind a feature toggle, and after talking to the benefits-dependents-management team, they verified that the modal direction that was previously in place was never launched to production, and that we could remove the modal/imports from the application. For further information on this, there is a [DSVA slack thread here](https://dsva.slack.com/archives/C0547Q0K0LF/p1698687571388759). I also removed the test around this modal as well since it is no longer needed.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/43066

## Testing done

- Removed old cypress test
- Removed app imports and tested application locally
- Insured all current unit tests pass, but no additional functionality was added

## Screenshots

No UI changes

## What areas of the site does it impact?

Profile / Personalization

## Acceptance criteria

- PT 1 cross app imports have been removed and everything still works

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed

### CODE REVIEW

**Review by dellerbie — APPROVED** [review]
:shipit:

File: src/applications/personalization/view-dependents/layouts/ViewDependentsLists.jsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ViewDependentsList from '../components/ViewDependentsList/ViewDependentsList';
-import DependencyVerificationModal from '../../../static-pages/dependency-verification/components/dependencyVerificationModal';
+// we commented out this import and implementation because the modal was deemed not ready for production
+// and we wanted to keep it in case work picked back up
+// commenting it out allows us to isolate the personalization app folder for ci/cd purposes
+// import DependencyVerificationModal from '../../../static-pages/dependency-verification/components/dependencyVerificationModal';
```
[the remark this answers is not in the document]
Comment (adamwhitlock1) [reply src/applications/personalization/view-dependents/layouts/ViewDependentsLists.jsx:6]: Yea you are right, and I prefer that as well. I think Ill just remove it.

**Review by Midge-dev — APPROVED** [review]
LGTM

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by allisonlu — APPROVED** [review]
lgtm!

**Pull request merged**